### PR TITLE
Document about support of multiple customer number on product in rest…

### PR DESCRIPTION
**Pull request by pankajagrawal16**
… request

### CODE REVIEW

**Comment by bringdev** [comment]
[trello.com/c/IYFIt4MH](https://trello.com/c/IYFIt4MH)

File: api/shipping-guide_2/topics.html [hunk 1]
```diff
@@ -24,122 +24,108 @@ <h1 class="api__title">Let's talk about Shipping Guide</h1>
...
+        <p align="justify">There is no home delivery on bank holidays. It can also happen that Bring will not do home delivery on certain days during Christmas or Easter. The service will not take this into account.
+          This must be handled by clients. End-users should not get delivery dates for holidays.</p>
         <p>TODO:</p>
       </section>
-      <section class="api__section">
 
+      <section class="api__section">
         <h2>More than one customer number with JSON</h2>
-        <p>TODO: Explain that it is not possible for now. Explain that it can be solved by doing multiple calls</p>
+        <p align="justify">This is now possible with version 2.0. Now we can request for multiple products with multiple customer number in single REST request similar to that SOAP version.
```
Comment (pankajagrawal16) [inline api/shipping-guide_2/topics.html]: Note only change here is this and rest is just alignment change

File: api/shipping-guide_2/api.raml [hunk 2]
```diff
@@ -304,6 +303,9 @@ documentation:
       topostalcode:
         description: Receiver postal code
         type: string
+      product:
+        description: List of products requested. Multiple products can be requested in a single request. Requested product can also be specified with a customer number separated with delimiter (**:**). For example CARGO**:**CARGO-123456
```
Comment (chege) [inline api/shipping-guide_2/api.raml]: Not sure if i understand `CARGO**:**CARGO-123456`. Do you mean `CARGO:CARGO-123456` ?

Comment (pankajagrawal16) [reply api/shipping-guide_2/api.raml]: Yes.. thats for making it bold on UI

Comment (pankajagrawal16) [reply api/shipping-guide_2/api.raml]: its not visible same way on UI

Comment (chege) [reply api/shipping-guide_2/api.raml]: hahaha...markdown?

Comment (pankajagrawal16) [reply api/shipping-guide_2/api.raml]: Yup

**Pull request merged**